smart-py/marketplace_tests.py

```diff
@@ -0,0 +1,870 @@
+import smartpy as sp
+import os
+
+Marketplace = sp.io.import_script_from_url(f"file://{os.getcwd()}/marketplace.py").Marketplace
+OBJKTSwapClass = sp.io.import_script_from_url(f"file://{os.getcwd()}/objkt_swap.py")
+
+@sp.add_test("Class constructs with default values")
+def test():
+    # init test and create html output
+    scenario = sp.test_scenario()
+    scenario.h1("Marketplace Test")
+
+    # init test values
+    objkt = sp.test_account("objkt123")
+    manager = sp.test_account("manager")
+    seller = sp.test_account("seller")
+
+    creator = sp.test_account("creator")
+    metadata = sp.record(
```

Comment:
Would it make sense to have a utility method that creates the default metadata since this is repeated across the tests?

Reply:
great idea